docs/setup.md

```diff
@@ -8,37 +8,59 @@ description: |-
 
 # Databricks-Sync
 
-Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+Databricks-Sync is a synchronization utility for the export and import of Databricks-native objects from one workspace to another. The primary use case for Databricks-Sync is Disaster Recovery (incremental backup and restore); however, it can be used as a one-time migration utility.
```

Comment:
I think that "one-time" migration utility gives a wrong impression.
IMO we should position the tool for both migration and DR. you can think about it as a continuous migration.

Should we mention the CI/CD option as well?

Reply:
```suggestion
Databricks-Sync is a synchronization utility for the export and import of Databricks-native objects from one workspace to another. The primary use cases for Databricks-Sync are Disaster Recovery, migration, and CI/CD.
```

Reply:
@itaiw - does this sound better?